README.md

```diff
@@ -51,10 +51,14 @@ Your feedback is more than welcome!
 * Check out [Open Issues](https://github.com/vmware-tanzu/antrea/issues).
 * Join the [Kubernetes Slack](http://slack.k8s.io/) and look for our
 [#antrea](https://kubernetes.slack.com/messages/CR2J23M0X) channel.
-* Attend the [project weekly meeting](https://VMware.zoom.us/j/823654111?pwd=MEV6blNtUUtqallVSkVFSGZtQ1kwUT09),
+* Check the [Antrea Team Calendar](https://calendar.google.com/calendar/embed?src=uuillgmcb1cu3rmv7r7jrhcrco%40group.calendar.google.com)
```

Comment:
when I click on the link I get: `Events from one or more calendars could not be shown here because you do not have the permission to view them.`

Reply:
i get the same error

Reply:
I gave permission to projectantrea-dev mailing list, let me know if you can access it.

Reply:
I made it public now. Can you please double-check that you can see it even when logged out of google (and that you cannot add/modify events)?

Reply:
I confirm that I can access the calendar when logged out

Reply:
@salv-orlando CI is failing because it thinks this is a dead link. Links look good to me, so it may be a general issue with Google calendar links. To bypass this, you could edit your PR to add the `"^https://calendar.google.com/*"` pattern to https://github.com/vmware-tanzu/antrea/blob/main/hack/.md_links_config.json